services/brig/src/Brig/API.hs

```diff
@@ -188,6 +217,10 @@ sitemap o = do
     Doc.errorResponse userNotFound
   ---
 
+  -- If the user is expired, it will be removed asynchronously.
+  -- This leads to the following events being sent to clients:
+  -- - UserDeleted event to contacts of the user
```

Comment:
This feels ... odd to me

Reply:
It is, but clients can create users with an `expires_at` field set (not sure in which case they do). I think we don't actively delete them through some kind of scheduled deletion or regular cleanup job, but just lazily when someone asks for them.

I will make clear that this is only about these ephemeral users and reference `Brig.API.User.userGC`.

Reply:
I think this is for guest accounts only.  

I wonder if we can get away with just using the TTL functionality of  Cassandra. But there's probably a good reason not to do that; I remember TIago telling me about this a few months ago